src/main/java/org/jenkinsci/plugins/docker/workflow/WithContainerStep.java

```diff
@@ -177,6 +177,11 @@ private static void destroy(String container, Launcher launcher, Node node, EnvV
             }
 
             container = dockerClient.run(env, step.image, step.args, ws, volumes, volumesFromContainers, envReduced, dockerClient.whoAmI(), /* expected to hang until killed */ "cat");
+            final List<String> ps = dockerClient.listProcess(env, container);
+            if (!ps.contains("cat")) {
+                listener.error("The container started but didn't run the expected command. Please double check your ENETRYPOINT does execute the command passed as docker run argument. See https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#entrypoint for entrypoint best practices.");
```

Comment:
Small typo here: ENETRYPOINT should be ENTRYPOINT

Reply:
Yup, this needs to be fixed. =)